### DISCUSSION TITLE
Copilot not working visual studio code

### DISCUSSION THREAD

**Discussion by Trigger-TG** [community/community#18429] [Copilot Conversations]
Yesterday i was using copilot in vsc just fine and today when i opened my vsc it didnt workt annymore, everytime i wrote a comment and then tabbed to get a suggestion nothing showed up. When i used the CTRL+ENTER command it showed my no suggestions or wrong ones. so i tried it with very simple code like adding 2 numbers, still didnt work. i tried everything i found on forums and reinstalled it many times but it still doesnt work.

**Comment by Trigger-TG** [comment]
> Yesterday i was using copilot in vsc just fine and today when i opened my vsc it didnt workt annymore, everytime i wrote a comment and then tabbed to get a suggestion nothing showed up. When i used the CTRL+ENTER command it showed my no suggestions or wrong ones. so i tried it with very simple code like adding 2 numbers, still didnt work. i tried everything i found on forums and reinstalled it many times but it still doesnt work.


Here is what is says in output
[INFO] [default] [2022-06-12T13:20:26.766Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-12T13:20:26.920Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 154 ms
[INFO] [streamChoices] [2022-06-12T13:20:26.920Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5IHnv9ybcbeePYUiS6hVVwQVvZWmH}] created: [{1655040027}]

**Comment by Zerotask** [comment]
It seems that other users here are having similar issues lately. Check if you are logged in for Copilot, everything is up to date and no other extension is blocking it
